app/views/home/lost_cookie.html.erb

```diff
@@ -2,4 +2,6 @@
 
 <div class="userstuff">
   <p><%= ts("You have been automatically logged out to protect your privacy.  Please make sure that your browser is set to accept cookies from archiveofourown.org and delete any existing Archive cookies.  If you continue to have problems logging in, please  <a href=\"#{new_feedback_report_url}\">contact Support</a>.".html_safe)  %></p>
+  <% page = params[:return_to_url].blank? ? root_path : params[:return_to_url] %>
+  <p><%= link_to(ts('Login returning to %{path}', path: page), new_user_session_path(return_to_url: page)) %></p>
```

Comment:
"Log in and return to %{path}" would be better link text, I think (we use "log in" for a verb and "login" for a noun or adjective). But possibly a silly question: is there a reason we can't include the login form itself on this page?